src/components/App/App.component.jsx

```diff
@@ -30,9 +53,17 @@ function App() {
     <BrowserRouter>
       <AuthProvider>
         <Layout>
+          <MyAppBar
+            query={query}
+            handleQuery={handleQuery}
+            handleKeyDown={handleKeyDown}
+          />
           <Switch>
             <Route exact path="/">
-              <HomePage />
+              <HomePage data={data} setVideo={setVideo} />
+            </Route>
+            <Route exact path="/video-detail">
+              <StyledVideoDetail video={video} setVideo={setVideo} />
```

Comment:
De momento está bien, pero como te dije, creo que eventualmente vamos a tener que cambiar este `setVideo` para obtener la info del video en el componente `VideoDetail` desde el url de la aplicación.

Si quieres implementarlo de una vez, sugiero que aquí hagas lo siguiente.

```suggestion
              <HomePage data={data} setVideo={setVideo} />
            </Route>
            <Route exact path="/video-detail/:videoId">
              <StyledVideoDetail video={video} setVideo={setVideo} />
```

De esa forma dentro de tu componente `VideoDetail` puedes hacer una llamada como 

```
import { useParams } from 'react-router-dom';
const params = useParams();
console.log(params.videoId) // => someRandomIDWeGotFromURL
```